espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/Location.kt

```diff
@@ -14,31 +14,6 @@
  * limitations under the License.
  */
 
-package io.appium.espressoserver.lib.model;
+package io.appium.espressoserver.lib.model
 
-import javax.annotation.Nullable;
-
-@SuppressWarnings("unused")
-public class Location extends AppiumParams {
-    private Integer x = null;
-    private Integer y = null;
-
-    @Nullable
-    public Integer getX() {
-        return x;
-    }
-
-    public void setX(Integer x) {
-        this.x = x;
-    }
-
-    @Nullable
-    public Integer getY() {
-        return y;
-    }
-
-    public void setY(Integer y) {
-        this.y = y;
-    }
-
-}
+data class Location(var x: Int? = null, var y: Int? = null) : AppiumParams()
```

Comment:
does it make any point to make properties optional? a location without any property set is invalid